src/composables/useQueryParams.js

```diff
@@ -42,6 +42,7 @@ export default function useQueryParams() {
     }
     let searchValue = filter_input.value
     if (
+      Array.isArray(filterOptions) &&
```

Comment:
If filterOptions is not an array what is it for the runs page? I think it should always be passed in as an array.

Reply:
It's slightly different, not sure why tho. Have a look at `src/views/pacbio/PacbioPoolIndex.vue` compared to `src/views/pacbio/PacbioRunIndex.vue`

Reply:
Ah. I dont think it needs to be a ref in PacbioRunIndex as its not dynamic so if you remove that does it work?

Reply:
seems to break the runs page. not sure if it's worth refactoring. what do you think?

Reply:
Sorry I meant remove the ref function wrapper but keep the filterOptions as a normal array.